Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -1324,6 +1324,7 @@ void ctkDICOMBrowser::exportSeries(QString dirPath, QStringList uids)
   foreach (const QString& uid, uids)
   {
     QStringList filesForSeries = d->DICOMDatabase->filesForSeries(uid);
+    filesForSeries.sort();
```

Comment:
Sorting by filenames is generally not the same as sorting by spatial position. In general, sorting is extremely complicated, because frames may need to be sorted based on different fields (e.g., they may all be in the same position and may need to be sorted based on various timestamps) and you may also need to use multiple fields for sorting (e.g., sort by cardiac phase and then sort based on spatial position).